docs/HowToUsePeripheral.md

```diff
@@ -131,3 +135,99 @@ static struct up_dev_s g_uart0priv = {
 ## I2C
 
 ## I2S
+
+## MPU
+* MPU enables you to partition memory into regions and set individual protection attributes for each region.
```

Comment:
These are basic content of MPU. This is a porting guide. Do we need to say this?
I want to know the meaning of functions.
mpu_priv_intsram_wb => what's the meaning of wb? what's the difference between intsram and intsram_wb?
mpu_priv_stronglyordered => what's the meaning of stonglyordered? when should we use this?
so on..

Reply:
will modify as per your suggestions